### DISCUSSION TITLE
default value of custom field not set when time entry is created via calendar view

### DISCUSSION THREAD

**Discussion by AnotherCoolDude** [kimai/kimai#3303] [General]
**Describe the bug**
I have declared a custom timesheet field, which is a list of items. The field is mandatory and has a default value. 
When I try to create a new timesheet entry using the calendar, the custom field is blank (instead of showing the default value). Once I open the newly created timesheet entry and save it, the default value is set correctly. This issue might be related to https://github.com/kevinpapst/kimai2/issues/2842

**To Reproduce**
1. Define a mandatory custom field with a default value for timesheets
2. Create a new timesheet entry via calendar (don't open and save it)
3. Export the timesheet as excel file 
4. The export shows blank fields instead of the default value

**Additional context**
 - Kimai version: 1.19.6
 - PHP version: 8.0.14
 - Device: Windows Desktop 27 inch
 - Browser Edge 101.0.1210.32

**Comment by kevinpapst** [comment]
It works for me here... how did you declare the additional field?

**Comment by AnotherCoolDude** [comment]
I declared the custom field like this
![custom_field](https://user-images.githubusercontent.com/11141338/167091451-ac5e58bc-8290-4ce0-851a-bd4ae250b0f5.jpg)

**Comment by kevinpapst** [comment]
Is that a dropdown?

**Comment by AnotherCoolDude** [comment]
It's a choice-list, I think that's what you mean by dropdown?
![custom_field_choice](https://user-images.githubusercontent.com/11141338/167163342-ffba83b3-fd62-4aab-938d-0aa38ec9c5d2.jpg)

**Comment by kevinpapst** [comment]
If you create a plain field (e.g a text field with a default value) it will be stored. 
But a choice field has no default value.
I will think about a solution, but maybe it would be better to create a checkbox "remote" or something alike.

**Comment by AnotherCoolDude** [comment]
I'll go with the checkbox, thank you very much for your support.
